src/azure-cli/azure/cli/command_modules/apim/commands.py

```diff
@@ -154,3 +165,15 @@ def load_command_table(self, _):
         g.custom_command('list', 'apim_ds_list')
         g.custom_show_command('show', 'apim_ds_get')
         g.custom_command('purge', 'apim_ds_purge')
+
+    with self.command_group('apim graphqlapi resolver', graphql_api_resolver) as g:
+        g.custom_command('create', 'apim_graphqlapi_resolver_create')
+        g.custom_command('delete', 'apim_graphqlapi_resolver_delete')
```

Comment:
Do we need to consider adding `confirmation=true` for these delete commands?

Reply:
added confirmation